test/assertions/to-have-items-satisfying.spec.js

```diff
@@ -186,17 +185,21 @@ describe('to have items satisfying assertion', () => {
   it('provides a detailed report of where failures occur', () => {
     expect(
       function() {
-        expect([0, 1, '2', 3, 4], 'to have items satisfying', function(item) {
-          expect(item, 'to be a number');
-          expect(item, 'to be less than', 4);
-        });
+        expect(
+          [0, 1, '2', 3, 4],
+          'to have items satisfying',
+          expect.it(function(item) {
+            expect(item, 'to be a number');
+            expect(item, 'to be less than', 4);
+          })
```

Comment:
Could also be written:

```js
expect(
  [0, 1, 2, 3, 4],
  'to have items satisfying',
  expect.it('to be a number').and('to be less than', 4)
)
```

Reply:
Ah nice, and it would also exercise the `.and()` path.